### PR TITLE
Solver dataset v2

### PR DESCRIPTION
**Pull request by tlegen-k**
## Related Issues and Dependencies
Closes Issue #34 
…

## This introduces a breaking change

- [ ] Yes
- [x ] No

## This Pull Request implements

Add Thoth Solver Dataset v2.0
Updated README.md
Add jupyter notebook to work with updated dataset

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
lgtm :+1: 
/lgtm

**Comment by tlegen-k** [comment]
With commit fde8d3314ad2c6395b40678eb03c1c503c0a796a I tried to suppress unnecessary output of cells in jupyter notebook.
It works locally or on cloud platform (https://colab.research.google.com), but on Github it doesn't collapse the output.
Please refer to lines with notebook metadata 100-102 in file notebooks/thoth-solver-dataset/ThothSolverDataset-v2.0.ipynb.

File: notebooks/thoth-solver-dataset/ThothSolverDataset-v2.0.ipynb [hunk 1]
```diff
@@ -97,7 +97,9 @@
    "cell_type": "code",
    "execution_count": 3,
    "id": "0611781e",
-   "metadata": {},
+   "metadata": {
```
Comment (tlegen-k) [inline notebooks/thoth-solver-dataset/ThothSolverDataset-v2.0.ipynb:100]: This metadata label seems not to work

Comment (tlegen-k) [reply notebooks/thoth-solver-dataset/ThothSolverDataset-v2.0.ipynb:100]: Fixed by 064a50618d695908d4dfb4469e016f51b5546af3.

**Comment by tlegen-k** [comment]
Hi @pacospace 
Removed unnecessary long output in commit 064a50618d695908d4dfb4469e016f51b5546af3.

**Review by pacospace — APPROVED** [review]
Thanks! 💯 lgtm!!

**Pull request merged**